addons/Yush/Yush.lua

```diff
@@ -75,12 +77,24 @@ output = function()
     if settings.Verbose then
         names[current] = names[current] or 'Unnamed ' .. tostring(current):sub(8)
 
+        output_text_table = {}
+        table.insert(output_text_table, names[current])
+        if settings.VerboseKeys then
+            for key, val in pairs(current) do
+                if type(val) ~= 'string' then
+                    val = names[val] or 'Unnamed'
```

Comment:
I don't have a strong feeling about this, but this could lead to several nested `Unnamed`s, which is a bit meaningless. Instead, you could do the same thing I do above and append `tostring(val):sub(8)`, so it's at least consistent. That said, you could also adjust my previous code and *remove* the `tostring(current):sub(8)` on line 78. Could also live with that, since I don't think appending a table identifier is of much use to anyone. In fact, I'd probably prefer this...